docs/modules/ROOT/pages/reference/extensions/management.adoc

```diff
@@ -33,6 +33,20 @@ ifeval::[{doc-show-user-guide-link} == true]
 Check the xref:user-guide/index.adoc[User guide] for more information about writing Camel Quarkus applications.
 endif::[]
 
+[id="extensions-management-usage"]
+== Usage
+For information on using Managed Beans in Camel, consult the https://camel.apache.org/manual/jmx.html[JMX section of the Camel Manual].
```

Comment:
Please replace this with an `xref` link. You can find examples in some of the other extension docs. Just grep for `xref:manual`.

Reply:
Done. I think I got the syntax right...